src/SDKs/Advisor/Management.Advisor/Microsoft.Azure.Management.Advisor.csproj

```diff
@@ -6,7 +6,7 @@
     <PropertyGroup>
         <Description>Microsoft Azure Advisor Library</Description>
         <AssemblyName>Microsoft.Azure.Management.Advisor</AssemblyName>
-        <Version>0.9.0-preview</Version>
+        <Version>0.9.5-preview</Version>
         <PackageId>Microsoft.Azure.Management.Advisor</PackageId>
         <PackageTags>Management.Advisor;Advisor;recommendations;</PackageTags>
         <PackageReleaseNotes>
```

Comment:
Please update `PackageReleaseNotes`

Reply:
Please create an `AssemblyInfo.cs` similar to [this](https://github.com/Azure/azure-sdk-for-net/tree/psSdkJson6/src/SDKs/Compute/Management.Compute/Properties) and set `AssemblyVersion` to `0.9.0.0` and `AssemblyFileVersion` to `0.9.5.0`

Reply:
@dsgouda Added AssemblyInfo.cs but did not update PackageReleaseNotes since functionally there is no change (just an additional property that gets exposed).